colin/checks/best_practices/help_file.py

```diff
@@ -0,0 +1,15 @@
+from colin.checks.abstract.filesystem import FileSystemCheck
+
+
+class HelpFileCheck(FileSystemCheck):
+
+    def __init__(self):
+        super().__init__(name="help_file_required",
+                         message="The 'helpfile' has to be provided.",
+                         description="Just like traditional packages, containers need "
+                                     "some 'man page' information about how they are to be used,"
+                                     " configured, and integrated into a larger stack.",
+                         reference_url="https://fedoraproject.org/wiki/Container:Guidelines#Help_File",
+                         files=['/help.1.', '/README.md'],
```

Comment:
Why `/help.1.`? It should be `/help.1`, right?

Reply:
Thanks!

Reply:
I've discovered, that Fedora requires one of the `help.1`/`README.md`, RHEL specifies only `help.1`.

I've created two different checks for each of them.